k8s-deploy/examples/party-10000/cluster-spark-local-pulsar.yaml

```diff
@@ -65,6 +65,8 @@ pulsar:
   type: NodePort
   httpNodePort: 30104
   httpsNodePort: 30109
+  env:
+  confs:
```

Comment:
In examples, we don't write useless stuffs. I suggest remove them and update the document:

https://github.com/FederatedAI/KubeFATE/blob/develop-1.9.0/docs/configurations/FATE_cluster_configuration.md

I find that this document has no configration information for pulsar. So please help to add all the pulsar related information into this file, thank you very much!!